products/ceqr_survey/models/staging/stg__dcm_arterial_highways.sql

```diff
@@ -1,12 +1,11 @@
 WITH arterial_highways_raw AS (
-
     SELECT * FROM {{ source('ceqr_survey_sources', 'dcm_arterial_highways') }}
-
 )
 
 SELECT
-    name,
-    wkb_geometry AS unbuffered_geom,
-    ST_BUFFER(wkb_geometry, 75) AS buffered_geom
+    "name" AS variable_id,
```

Comment:
does "name" have to be in quotes?

Reply:
nope. and neither does source. Removed them (can't recall why they were there)

Reply:
> nope. and neither does source. Removed them (can't recall why they were there)

pluto 2.0 🥲